charts/ditto/templates/_helpers.tpl

```diff
@@ -0,0 +1,55 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "eclipse-ditto.name" -}}
```

Comment:
Usually the properties are named as the chart, means `ditto` i.s.o. `eclipse-ditto`.